packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -236,58 +243,14 @@ export class TransactionPool {
 		this._queues.validated.enqueueMany(removedTransactionsBySenderPublicKeys);
 	}
 
-	public async validateReceivedTransactions(): Promise<
-		ReadonlyArray<TransactionResponse>
-	> {
-		// Get transactions from the received queue
-		const transactionsToValidate = this._queues.received.peekUntil(
-			queueCheckers.returnTrueUntilLimit(
-				this._receivedTransactionsProcessingLimitPerInterval,
-			),
-		);
-		// Validate transactions
-		const {
-			transactionsResponses: validateTransactionsResponse,
-		} = await this._validateTransactions(transactionsToValidate);
-
-		// Get ids of invalid transactions from validateTransactionsResponse
-		const invalidTransactionIds = validateTransactionsResponse
-			.filter(transactionResponse => transactionResponse.status === Status.FAIL)
-			.map(transationStatus => transationStatus.id);
-
-		// Filter transactions in transactionToValidate which are invalid
-		const invalidTransactions = transactionsToValidate.filter(transaction =>
-			invalidTransactionIds.includes(transaction.id),
-		);
-		// Filter transactions in transactionToValidate which are valid
-		const validTransactions = transactionsToValidate.filter(
-			transaction => !invalidTransactionIds.includes(transaction.id),
-		);
-		// Remove invalid transactions
-		this.queues.received.removeFor(
-			queueCheckers.checkTransactionForId(invalidTransactions),
-		);
-		// Move valid transactions from the received queue to the validated queue
-		this.queues.validated.enqueueMany(
-			this.queues.received.removeFor(
-				queueCheckers.checkTransactionForId(validTransactions),
-			),
-		);
-
-		return validateTransactionsResponse;
-	}
-
 	public validateTransactionAgainstTransactionsInPool(
 		transaction: Transaction,
 	): boolean {
-		// TODO: remove the condition for checking `verifyTransactionAgainstOtherTransactions` exists, because it should always exist
-		return transaction.verifyTransactionAgainstOtherTransactions
-			? transaction.verifyTransactionAgainstOtherTransactions([
-					...this.queues.ready.transactions,
-					...this.queues.pending.transactions,
-					...this.queues.verified.transactions,
-			  ])
-			: true;
+		return transaction.verifyTransactionAgainstOtherTransactions([
```

Comment:
in transactions lib it is named `verifyAgainstOtherTransactions`. the extra `Transaction` seems unneeded